js/components/unmuteIcon.js

```diff
@@ -13,9 +13,14 @@ var UnmuteIcon = createReactClass({
   },
 
   unmuteClick: function(event) {
+    event.stopPropagation();
     this.props.controller.handleMuteClick();
   },
 
+  onMouseUp: function(event) {
```

Comment:
This is needed because the ad screen uses the on `mouseup` event for the clickthrough.

Reply:
May I ask you to put this note into comments?